diff --git i/code/AssetLib/USD/USDLoaderImplTinyusdz.cpp w/code/AssetLib/USD/USDLoaderImplTinyusdz.cpp
index 31c2463d7..e5fd307a0 100644
--- i/code/AssetLib/USD/USDLoaderImplTinyusdz.cpp
+++ w/code/AssetLib/USD/USDLoaderImplTinyusdz.cpp
@@ -73,7 +73,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "../../../contrib/tinyusdz/assimp_tinyusdz_logging.inc"
 
 namespace {
-    static constexpr char TAG[] = "tinyusdz loader";
+    // static constexpr char TAG[] = "tinyusdz loader";
 }
 
 namespace Assimp {
